Production/crab/config/2016/Graviton.txt

```diff
@@ -1,19 +1,19 @@
 EventAwareLumiBased
-globalTag=80X_mcRun2_asymptotic_2016_TrancheIV_v8 sampleType=Summer16MC applyTriggerMatch=True saveGenTopInfo=False saveGenBosonInfo=True saveGenJetInfo=True
+globalTag=94X_mcRun2_asymptotic sampleType=Summer16MC applyTriggerMatch=True saveGenTopInfo=False saveGenBosonInfo=True saveGenJetInfo=True
```

Comment:
for the signals (graviton, radion, nonres) you should store gen info and avoid applying cut on the trigger:
`applyTriggerCut=False storeLHEinfo=True saveGenParticleInfo=True`